tests/testsuite/generate_lockfile.rs

```diff
@@ -215,6 +223,7 @@ fn duplicate_entries_in_lockfile() {
             name = "b"
             authors = []
             version = "0.0.1"
+            edition = "2015"
```

Comment:
Can this be split out into a separate commit?

Reply:
done!